relion/protocols/protocol_extract_particles.py

```diff
@@ -169,7 +169,7 @@ def _extractMicrograph(self, mic, params):
         self._extractMicrographList([mic], params)
 
     def _extractMicrographList(self, micList, params):
-        if not micList:  # do not process when empty list, need to check this properly
+        if len(micList) <= 0:  # do not process when empty list, need to check this properly
```

Comment:
@delarosatrevin how len can be < 0?

Reply:
I'm not sure about this change (maybe I was drunk or something), but it was failing. Well, the boolean check might be sometimes tricky since it can be re-defined by a class. So, a "set-like" class could test True as boolean and be empty.